bus-mapping/src/rpc.rs

```diff
@@ -207,16 +262,16 @@ mod rpc_tests {
     #[ignore]
     #[tokio::test]
     async fn test_trace_block_by_number() {
-        let transport = Http::new(Url::parse("http://localhost:8545").unwrap());
-        let prov = GethClient::new(transport);
-        let trace_by_hash = prov.trace_block_by_number(5.into()).await.unwrap();
+        let prov = get_provider();
+        let trace_by_hash = prov.trace_block_by_number(2.into()).await.unwrap();
```

Comment:
2 should not be hardcoded here. Same as above.

Reply:
Okay!